src/api/lib/delayed_job_monitor.rb

```diff
@@ -0,0 +1,18 @@
+class DelayedJobMonitor
+  def run
+    send_email if delayed_jobs_warning_limit && Delayed::Job.count >= delayed_jobs_warning_limit
+  end
+
+  private
+
+  def send_email
+    message = "The number of delayed jobs in the queue has reached #{delayed_jobs_warning_limit}! Please check that delayed job is running."
+    AdminMailer.warning(message).deliver_now
```

Comment:
The AdminMailer already exists, right?
Maybe we should introduce a StaffMailer or extend the AdminMailer to also include user with role staff (-> our dev team has the role staff on build.o.o)